src/ansys/dpf/core/operators/logic/elementary_data_selector.py

```diff
@@ -90,7 +100,8 @@ def _spec():
                     optional=False,
                     document="""One or several elementary data index that
```

Comment:
Use same description as earlier. We basically want clear, complete sentences in the present tense.